modules/travel_pay/app/services/travel_pay/documents_service.rb

```diff
@@ -25,9 +25,9 @@ def download_document(claim_id, doc_id)
 
       # The content type comes across as the type of the binary data,
       # but the actual content type of the response is application/json
-
+      # so we need to parse the response body in order to access the data
       {
-        body: response.body['data'],
+        body: JSON.parse(response.body)['data'],
```

Comment:
@seesleestak I added this to this PR to avoid another 3-day wait. This fixes a document download bug. Let me know if you want to chat about this